server/config/requestHandler.js

```diff
@@ -49,4 +49,18 @@ module.exports = {
     res.send(responseJSON);
   },
 
-};
\ No newline at end of file
+  getPhotos: function(req, res) {
+    var responseJSON = {
+      photos: Photos
+    }
+
+    res.send(responseJSON);
+  },
+
+  postPhotos: function(req, res) {
+    var photo = req.body
+    Photos.push(photo);
```

Comment:
I know the Photos array is a temporary thing, but shouldn't we be storing is with some sort of User designation so that we can understand who it is associated with?... I could just be misunderstanding what you're trying to do here.